components/account/AccountRelationshipIndicator.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+import type { mastodon } from 'masto'
+
+defineProps<{
+  relationship?: mastodon.v1.Relationship
```

Comment:
IMO this shouldn't be optional and rather the v-if should be applied outside of this component

Reply:
Thanks for your review. That makes sense. I changed it to be non-optional.